roles/restart_proxy/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
I wonder if this should be its own role. or if this should be in the ```ruby_libvirt_gem``` role as a trigger so it only restarts when it's installed.